privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -817,11 +818,11 @@ def loadConfig(self, config):
         self.tls_verify = is_true(config.get("TLS_VERIFY", False))
         # Fallback to DEFAULT_TLS_PROTOCOL (TLSv1: 3, TLSv1.1: 4, v1.2: 5, TLS negotiation: 2)
         self.tls_version = int(config.get("TLS_VERSION") or DEFAULT_TLS_PROTOCOL)
-        self.tls_ca_file = config.get("TLS_CA_FILE") or DEFAULT_CA_FILE
```

Comment:
Why remove the default file? (See the definition at the beginning of this file)

Reply:
The default file will be set in `_get_tls_context()` here:
https://github.com/privacyidea/privacyidea/blob/02f9e89507416e90369da6f5259729935624a834/privacyidea/lib/resolvers/LDAPIdResolver.py#L526
This is the same behavior as with `testconnection()` when no `TLS_CA_FILE` is set in the LDAP configuration.